qtpy/tests/test_qtmultimediawidgets.py

```diff
@@ -6,15 +6,11 @@
 from qtpy.tests.utils import using_conda
 
 
-@pytest.mark.skipif(not (PYQT5 or PYSIDE2), reason="Only available in Qt5 bindings")
-@pytest.mark.skipif(
-    using_conda(), reason="Conda packages don't seem to include QtMultimedia"
-)
 def test_qtmultimediawidgets():
     """Test the qtpy.QtMultimediaWidgets namespace"""
     from qtpy import QtMultimediaWidgets
 
-    assert QtMultimediaWidgets.QCameraViewfinder is not None
+    # assert QtMultimediaWidgets.QCameraViewfinder is not None
```

Comment:
I think it would be better to add a validation to test this instead of just commenting the assert, something like:

```python
if QT5:    # Only available for Qt5 bindings
    assert QtMultimediaWidgets.QCameraViewfinder is not None
```
Probably the `QtMultimediaWidgets.QVideoWidgetControl` was commented for that reason and we could move it to the Qt5 validation?

Reply:
actually QVideoWidgetControl seems not available with pyqt5